server/pkg/property/initializer.go

```diff
@@ -1,4 +1,4 @@
-//go:generate go run github.com/globusdigital/deep-copy --type Initializer --pointer-receiver -o initializer_gen.go .
```

Comment:
[memo]
when directly we use `github.com/globusdigital/deep-cop`, go:generate comment is like this.
```
// generated by /var/folders/sd/pfjp2mqd1cs6hd_vj09vjnl00000gn/T/go-build811440719/b001/exe/deep-copy --type Initializer --pointer-receiver -o initializer_gen.go .; DO NOT EDIT.
```
Every time we run it, the `go-build811440719` part changes.